src/index.js

```diff
@@ -127,14 +127,12 @@ export default declare(api => {
 						},
 						this
 					);
+					for (const global of this.globals.reverse())
```

Comment:
fixes a problem with imports not able to be modified by other plugins, meaning output was a mix of cjs and es2015